doc/modules/clustering.rst

```diff
@@ -338,15 +338,17 @@ to be the exemplar of sample :math:`i` is given by:
 
 .. math::
 
-    r(i, k) \leftarrow s(i, k) - max [ a(i, \acute{k}) + s(i, \acute{k}) \forall \acute{k} \neq k ]
+    r(i, k) \leftarrow s(i, k) - \max_{\overline k \ne k} \left( a(i, \overline
```

Comment:
I've never seen acute's used like that. is that a French thing? I would usually use ``'`` (k prime)